doc.go

```diff
@@ -455,6 +455,41 @@ of the returned value:
 	    }
 	}
 
+## Arrow batches
+
+There is a possibility to retrieve data in a columnar format similar to native server response.
+You have to use a proper `WithArrowBatches` context, example:
+
+	    var rows driver.Rows
+		err = conn.Raw(func(x interface{}) error {
+			rows, err = x.(driver.QueryerContext).QueryContext(ctx, query, nil)
+			return err
+		})
+
+		...
+
+		batches, err := rows.(sf.SnowflakeRows).GetArrowBatches()
+
+		... // use arrow records
+
+Limitations:
+
+1. When a query regards something that is not real data, it is returned usually in JSON format. It can't be used with arrow batches context. Examples: `SHOW PARAMETERS`.
```

Comment:
Not sure what you are trying to say in the first sentence (not real data?).

When a query returns information instead of data, such as a SHOW PARAMETERS query, the information is usually returned as JSON. You cannot use JSON with arrow batches context.

Reply:
Wasn't sure how to call it. `SHOW PARAMETERS` returns some response, but this is not customer data, this is just a technical response. Maybe I can rephrase it with:

For some queries Snowflake may decide to return data in JSON format (examples: `SHOW PARAMETERS` or `ls @stage`). You cannot use JSON with Arrow batches context.

What do you think?